tutorials/01-Introduction.ipynb

```diff
@@ -0,0 +1,2872 @@
+   "id": "e31ae6c2-c08f-4cd9-95dc-6cbc0b060e18",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Integers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c7279349-50e6-4747-93b0-1fb5c6764de7",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "In many computer languages you may have learned through education (C, JAVA, ...), integers are not infinite and prone to overflow errors.\n",
+    "\n",
+    "In Python and SymPy, integers are really $\\mathbb{Z}$ and doesn't have problem with overflows, which allows for you to work with very large numbers, especially powers and factorials."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9e1a5db4-914c-4e13-9993-7218f6da3b0f",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "### Creating Integers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4be225fc-3caa-482e-a9f6-ac9cab144be6",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "You can create SymPy integer by passing python `int` to `Integer` constructor."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "id": "8fc620c1-954a-4a11-853e-89be4fe7a28a",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle 12345$"
+      ],
+      "text/plain": [
+       "12345"
+      ]
+     },
+     "execution_count": 58,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Integer(12345)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3a612d3-b27f-4940-926a-97e6abc89996",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "### Integer Arithmetic"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "626c8849-9f01-4095-ac0f-dc898a8e348e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "You can add, subtract, multiply, power, divide like other integer numbers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 63,
+   "id": "5b00f2e9-27c7-4274-92e1-abfdfa95ba1e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle 46$"
+      ],
+      "text/plain": [
+       "46"
+      ]
+     },
+     "execution_count": 63,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Integer(12) + Integer(34)"
```

Comment:
It's not necessary for SymPy users to ever use the Integer constructor, since integers will automatically be converted to SymPy integers when combined with SymPy expressions.

Reply:
Unfortunately, it is quite not trivial to explain how such things work 'automatically'.
I'm aware some sympy expressions cannot be constructed properly without constructing integers directly, because of division gotcha.